test/testcafe/spec/ChallengeAuthenticatorOktaVerify_spec.js

```diff
@@ -15,116 +22,120 @@ const requestLogger = RequestLogger(/challenge/,
-      'methodType': 'signed_nonce'
+      'methodType': 'push'
     },
     'stateHandle': '02ciZ1YTWakSanNu8GNNTnTXzhL5hoLzTlR0JewfG3'
   });
 });
 
-test.requestHooks(requestLogger, mockChallengeOVSelectMethod)('should send right methodType when push is selected', async t => {
+test.requestHooks(requestLogger, mockChallengeOVSelectMethod)('should send right methodType ' +
+  'when totp is selected', async t => {
   const selectAuthenticatorPage = await setup(t);
-  await t.expect(selectAuthenticatorPage.getFormTitle()).eql('Verify it\'s you with an authenticator');
-  await t.expect(selectAuthenticatorPage.getFormSubtitle()).eql('Select from the following options');
-  selectAuthenticatorPage.selectFactorByIndex(1);
-  const successPage = new SuccessPageObject(t);
-  const pageUrl = await successPage.getPageUrl();
-  await t.expect(pageUrl)
-    .eql('http://localhost:3000/app/UserHome?stateToken=mockedStateToken123');
-  const req2 = requestLogger.requests[0].request;
-  await t.expect(req2.url).eql('http://localhost:3000/idp/idx/challenge');
-  await t.expect(req2.method).eql('post');
-  await t.expect(JSON.parse(req2.body)).eql({
-    'authenticator':
-    {
+
+  await selectAuthenticatorPage.selectFactorByIndex(1);
+  await verifySelectAuthenticator(t, selectAuthenticatorPage, {
+    'authenticator': {
       'id': 'aut13qrZReYpIib7R0g4',
-      'methodType': 'push'
+      'methodType': 'totp'
     },
     'stateHandle': '02ciZ1YTWakSanNu8GNNTnTXzhL5hoLzTlR0JewfG3'
   });
 });
 
-test.requestHooks(requestLogger, mockChallengeOVSelectMethod)('should send right methodType when totp is selected', async t => {
+test.requestHooks(requestLogger, mockChallengeOVSelectMethod)('should send right methodType ' +
+  'when fastpass is selected', async t => {
   const selectAuthenticatorPage = await setup(t);
-  await t.expect(selectAuthenticatorPage.getFormTitle()).eql('Verify it\'s you with an authenticator');
-  await t.expect(selectAuthenticatorPage.getFormSubtitle()).eql('Select from the following options');
-  selectAuthenticatorPage.selectFactorByIndex(2);
-  const successPage = new SuccessPageObject(t);
-  const pageUrl = await successPage.getPageUrl();
-  await t.expect(pageUrl)
-    .eql('http://localhost:3000/app/UserHome?stateToken=mockedStateToken123');
-  const req2 = requestLogger.requests[0].request;
-  await t.expect(req2.url).eql('http://localhost:3000/idp/idx/challenge');
-  await t.expect(req2.method).eql('post');
-  await t.expect(JSON.parse(req2.body)).eql({
-    'authenticator':
-    {
+
+  await selectAuthenticatorPage.selectFactorByIndex(2);
+  await verifySelectAuthenticator(t, selectAuthenticatorPage, {
+    'authenticator': {
       'id': 'aut13qrZReYpIib7R0g4',
-      'methodType': 'totp'
+      'methodType': 'signed_nonce'
     },
     'stateHandle': '02ciZ1YTWakSanNu8GNNTnTXzhL5hoLzTlR0JewfG3'
   });
 });
+
+test.requestHooks(mockChallengeOVOnlyMethodsWithDeviceKnown)('the FastPass option is always on ' +
```

Comment:
nit: `'FastPass option is rendered 1st in the list when deviceKnown=true'`

Reply:
I updated the name